test-lib/test.js

```diff
@@ -1,15 +1,25 @@
 const fs = require('fs-extra');
 const path = require('path');
+const { stripIndent } = require('common-tags');
 
-if (!fs.existsSync(path.join(__dirname, '/../test/node_modules'))) {
-  fs.mkdirSync(path.join(__dirname, '/../test/node_modules'));
-  fs.symlinkSync(path.join(__dirname, '/..'), path.join(__dirname, '/../test/node_modules/apostrophe'), 'dir');
-}
+fs.removeSync(path.join(__dirname, '/../test/node_modules'));
+fs.mkdirSync(path.join(__dirname, '/../test/node_modules'));
+fs.symlinkSync(path.join(__dirname, '/..'), path.join(__dirname, '/../test/node_modules/apostrophe'), 'dir');
```

Comment:
The tests need to see themselves as apostrophe apps, so they need to see apostrophe itself as a loadable dependency which itself is treated as a bundle.